src/azure-cli/azure/cli/command_modules/appconfig/_constants.py

```diff
@@ -72,3 +72,8 @@ class KVSetConstants:
 class ImportExportProfiles:
     KVSET = "appconfig/kvset"
     DEFAULT = "appconfig/default"
+
+class ImportExportDiffKeys:
```

Comment:
We are using 3 forms of the verbs here: 
- `update` is returned by differ output
- `updated`  is used as an intermediate key
- `Updating` is what the user sees.

We can't change the first one, and the last one makes more sense to users. So can we use either of these forms for the intermediate dict key?
My preference is change it to `update' and the class can be called JsonDiffResult.
So the usage would look like this:
```
if action.label == JsonDiffResult.UPDATE:
            ff_diff[JsonDiffResult.UPDATE] = []
```

Reply:
Or just `JsonDiff.Update`

Reply:
Sounds good! This simplifies it. This has been updated. Thanks!